tex/generic/pgf/frontendlayer/tikz/tikz.code.tex

```diff
@@ -2173,6 +2173,7 @@
       \fi%
     \fi%
   \fi%
+  \ifx\pgfutil@next\tikz@handle@more\else\tikz@expandcount=100\relax\fi
```

Comment:
I've changed `\tikz@handle@more` to `\tikz@expand` here.  I don't see why the count should be reset in the case of `\tikz@handle@more`.

Reply:
No it should be `\tikz@handle@more` here. At the end of `\tikz@handle`, `\pgfutil@next` is either one of the handlers or `\tikz@handle@more`, but is always not `\tikz@expand`. The logic here is to reset the count when `\pgf@let@token` _is_ handled in `\tikz@handle`. Using `\pgfutil@ifx\pgfutil@next\tikz@expand` will reset the count even when `\pgf@let@token` is eventually not handled in `\tikz@handle@more`.